src/core/CCList.mli

```diff
@@ -111,73 +112,78 @@ val count : ('a -> bool) -> 'a list -> int
-(** Like {!List.combine} but tail-recursive.
-    Transform a pair of lists into a list of pairs:
-    [combine [a1; ...; an] [b1; ...; bn]] is
-    [[(a1,b1); ...; (an,bn)]].
+(** [combine [a1; ⋯; an] [b1; ⋯; bn]] is [[(a1,b1); ⋯; (an,bn)]].
+    Transform two lists into a list of pairs.
+    Like {!List.combine} but tail-recursive.
     @raise Invalid_argument if the lists have distinct lengths.
     @since 1.2, but only
     @since 2.2 with labels *)
 
 val combine_gen : 'a list -> 'b list -> ('a * 'b) gen
-(** Lazy version of {!combine}.
+(** [combine_gen l1 l2] transforms two lists into a [gen] of pairs.
+    Lazy version of {!combine}.
     Unlike {!combine}, it does not fail if the lists have different
     lengths;
     instead, the output has as many pairs as the smallest input list.
     @since 1.2, but only
     @since 2.2 with labels *)
 
 val split : ('a * 'b) t -> 'a t * 'b t
-(** A tail-recursive version of {!List.split}.
-    Transform a list of pairs into a pair of lists:
-    [split [(a1,b1); ...; (an,bn)]] is [([a1; ...; an], [b1; ...; bn])].
+(** [split [(a1,b1); ⋯; (an,bn)]] is [([a1; ⋯; an], [b1; ⋯; bn])].
+    Transform a list of pairs into a pair of lists.
+    A tail-recursive version of {!List.split}.
     @since 1.2, but only
     @since 2.2 with labels *)
 
 val compare : ('a -> 'a -> int) -> 'a t -> 'a t -> int
+(** [compare cmp l1 l2] compares the two lists [l1] and [l2]
+    using the given comparison function [cmp]. *)
 
 val compare_lengths : 'a t -> 'b t -> int
-(** Equivalent to [compare (length l1) (length l2)] but more efficient.
-    Compare the lengths of two lists.
+(** [compare_lengths l1 l2] compare the lengths of the two lists [l1] and [l2].
+    Equivalent to [compare (length l1) (length l2)] but more efficient.
     @since 1.5, but only
     @since 2.2 with labels *)
 
 val compare_length_with : 'a t -> int -> int
-(** Equivalent to [compare (length l) x] but more efficient.
-    Compare the length of a list to an integer.
+(** [compare_length_with l x] compares the length of the list [l] to an integer [x].
+    Equivalent to [compare (length l) x] but more efficient.
     @since 1.5, but only
     @since 2.2 with labels *)
 
 val equal : ('a -> 'a -> bool) -> 'a t -> 'a t -> bool
+(** [equal p l1 l2] returns [true] if [l1] and [l2] are equal. *)
 
 val flat_map : ('a -> 'b t) -> 'a t -> 'b t
-(** Map and flatten at the same time (safe). Evaluation order is not guaranteed. *)
+(** [flat_map f l] maps and flattens at the same time (safe). Evaluation order is not guaranteed. *)
 
 val flat_map_i : (int -> 'a -> 'b t) -> 'a t -> 'b t
-(** Map with index and flatten at the same time (safe).
+(** [flat_map_i f l] maps with index and flattens at the same time (safe).
     Evaluation order is not guaranteed.
-    @since 2.8
-*)
+    @since 2.8 *)
 
 val flatten : 'a t t -> 'a t
-(** Safe flatten. Concatenate a list of lists. *)
+(** [flatten [l1]; [l2]; ⋯] concatenates a list of lists. Safe flatten. *)
```

Comment:
Maybe you can use the same notation as before `Safe version of {!List.flatten}`

Reply:
OK.